### PR TITLE
Harr'qac expansion

### PR DESCRIPTION
**Pull request by samo-novak**
Expanding the Harr'qac colony. Fixed stuff. Feyksed gremmaire. Elaborated a lot of stuff. Created a lot of stuff.

I'm not finished yet, but I want the content I already have to be accessible already; I will keep on expanding Harr'qac further, it's just that I've had a lot of stuff and the only person that could see it was me - for a while.

@ParadoxonKomplikon should maybe have a look. Reviews welcome. Also from anyone else.

### CODE REVIEW

File: Planets and Systems/Harr'qac.md [hunk 1]
```diff
@@ -36,52 +36,103 @@
...
 
 ## Government
 
-Similliarly to Qerr'Balak, there are many city-states governing themselves. However, when colonising the planet, Skrell chose to try a similliar yet new form of government. City-states on each continent are grouped together. Qerr-Skria of these city-states elect a representative of their continent, called Qerr. This way, cities manage themselves, they decide about continent-wide things together and are represented by one Skrell for each continent when discussing global decisions. The three Qerrs meet in Government District of the University of Harr'qac.
+Similliarly to Qerr'Balak, there are many city-states governing themselves. However, when colonising the planet, Skrell chose to try a similliar, yet new form of government. City-states on each continent are grouped together. Qerr-Skria of these city-states elect a representative of their continent, called Qerr. This way, cities manage themselves, they decide about continent-wide things together and are represented by one Skrell for each continent when discussing global decisions. The three Qerrs meet in Government District of the University of Harr'qac and are very respected amongst the population.
```
Comment (ParadoxonKomplikon) [inline Planets and Systems/Harr'qac.md]: I don't think I've ever noticed that, so: You might want to use another word than "Qerr" for your representatives, as it's just a prefix generally attached to other words and it seems a bit weird to have it around on its own :p

**Comment by ParadoxonKomplikon** [comment]
Otherwise I don't have any lore issues with your additions, so that's a 👍 from me.

File: Planets and Systems/Harr'qac.md (hunk 1 above)
Comment (samo-novak) [reply Planets and Systems/Harr'qac.md]: Sure, I'm going to invent something new. Perhaps I'll use this prefix, not sure yet.

**Comment by samo-novak** [comment]
Renamed to Qerr-Kirri, if that's okay with you.
@ParadoxonKomplikon

**Comment by ParadoxonKomplikon** [comment]
All good !

**Comment by Raptor1628** [comment]
Will read this over for grammar later today.

File: Planets and Systems/Harr'qac.md [hunk 2]
```diff
@@ -36,52 +36,103 @@
...
 
 Same as on Qerrbalak, each city state also has its own academy.
 
 ## Economy
 
-Harr'qac is considered quite rich colony, with very good general living standard.
+Harr'qac is considered quite rich colony, with very good general living standard. 
+
+The economical system is basically identical to that of Qerrbalak - money is created for a finite period, after which is loses value - because there wasn't a reason to change this. However, some city states take a more direct approach to the human currency problem: Kaux-Nira and Qal-Si, both being major trading locations and the latter also having a lot of criminality, are currently keeping track of any human money that enters the cities - citizens of those two cities are obliged to annually report the amount on human currencies in their possession. They also heavily encourage their citizens to exchange the human money for Skrellian money. Some other city states are calling to their respective Qerr-Kirris for advice. This issue is currently a reason for a lot of high priority sessions in the Government District of the University of Harr'qac and many researchers, managers and diplomats are trying to find a viable solution.
```
Comment (Raptor1628) [inline Planets and Systems/Harr'qac.md]: *economic system

File: Planets and Systems/Harr'qac.md [hunk 3]
```diff
@@ -36,52 +36,103 @@
...
 
-The most famous academy of Harr'qac is The University of Harr'qac. It's not located any any city, but it rather creates a city like any other itself. It's located on the island of Korm. The university takes almost whole island, with lots of parks, services, leisure areas... It is supported by other cities, as the global government meets there.
+The most famous academy of Harr'qac is The University of Harr'qac. It's not located in any city, but it rather creates a city like any other itself. It's located on the island of Korm. The university takes almost whole island, with lots of parks, services, leisure areas and the like. It is supported by other cities, as the global government meets there.
 
 Same as on Qerrbalak, each city state also has its own academy.
 
 ## Economy
 
-Harr'qac is considered quite rich colony, with very good general living standard.
+Harr'qac is considered quite rich colony, with very good general living standard.
```
Comment (Raptor1628) [inline Planets and Systems/Harr'qac.md]: *is considered a wealthy colony, with an excellent standard of living.

File: Planets and Systems/Harr'qac.md [hunk 4]
```diff
@@ -36,52 +36,103 @@
...
 
 ## Education
 
-The most famous academy of Harr'qac is The University of Harr'qac. It's not located any any city, but it rather creates a city like any other itself. It's located on the island of Korm. The university takes almost whole island, with lots of parks, services, leisure areas... It is supported by other cities, as the global government meets there.
+The most famous academy of Harr'qac is The University of Harr'qac. It's not located in any city, but it rather creates a city like any other itself. It's located on the island of Korm. The university takes almost whole island, with lots of parks, services, leisure areas and the like. It is supported by other cities, as the global government meets there.
```
Comment (Raptor1628) [inline Planets and Systems/Harr'qac.md]: It was not originally located within any city, but one has built up around it over time, as is typical for schools of higher learning.

File: Planets and Systems/Harr'qac.md [hunk 5]
```diff
@@ -36,52 +36,103 @@
...
 
 ## Government
 
-Similliarly to Qerr'Balak, there are many city-states governing themselves. However, when colonising the planet, Skrell chose to try a similliar yet new form of government. City-states on each continent are grouped together. Qerr-Skria of these city-states elect a representative of their continent, called Qerr. This way, cities manage themselves, they decide about continent-wide things together and are represented by one Skrell for each continent when discussing global decisions. The three Qerrs meet in Government District of the University of Harr'qac.
+Similliarly to Qerrbalak, there are many city-states governing themselves. However, when colonising the planet, Skrell chose to try a similliar, yet new form of government. City-states on each continent are grouped together. Qerr-Skria of these city-states elect a representative of their continent, called Qerr-Kirri. This way, cities manage themselves, they decide about continent-wide things together and are represented by one Skrell for each continent when discussing global decisions. The three Qerr-Kirris meet in Government District of the University of Harr'qac and are very respected amongst the population.
```
Comment (Raptor1628) [inline Planets and Systems/Harr'qac.md]: *try a similar form of government, with some novel additions.

File: Planets and Systems/Harr'qac.md [hunk 6]
```diff
@@ -36,52 +36,103 @@
...
 
 ### Family life
 
-Family life on Harr'qac is similliar to the one on Qerr'Balak. Harr'qacian Skrell tend to bond fewer times in life. In some families, there is a recent tendency to put Qrri'Baqa into more contact with each other to strenghten their familial relations.
+Family life on Harr'qac is similliar to the one on Qerrbalak. Harr'qaci Skrell tend to bond fewer times in life. In some families, there is a recent tendency to put Qrri'Baqa into more contact with each other to strenghten their familial relations.
 
 ## Cities
 
-Harr'qaci Skrell have built cities, that, unlike on Qerrbalak, are not standing on stilts, but instead are directly on the ground. The cities are inside very solid glass spheres.
+Harr'qaci Skrell have built cities that, unlike on Qerrbalak, are not standing on stilts, but instead are directly on the ground. The cities are inside very resistant glass spheres.
```
Comment (Raptor1628) [inline Planets and Systems/Harr'qac.md]: *inside durable glass spheres. (would domes be better here?)

**Comment by Raptor1628** [comment]
Generally like it, just see to the grammar and spelling and I'll merge it.

File: Planets and Systems/Harr'qac.md (hunk 4 above)
Comment (samo-novak) [reply Planets and Systems/Harr'qac.md]: Not sure about this one. The "over time" part doesn't seem right, it was built this way intentionally - it was built on an uninhabited island, after all. I'm not sure though.

File: Planets and Systems/Harr'qac.md (hunk 6 above)
Comment (samo-novak) [reply Planets and Systems/Harr'qac.md]: Yeah, domes are probably what I meant.

File: Planets and Systems/Harr'qac.md (hunk 4 above)
Comment (samo-novak) [reply Planets and Systems/Harr'qac.md]: How about this:
It was not originally located within any city, but one was built up around it to facilitate its needs.

Although, the city part also includes the various areas and buildings used for education, not only the services and facilities for living.

@Raptor1628

**Comment by samo-novak** [comment]
Edited everything

The University of Harr'qac part of it being a city itself needs further review. Refer to the comments for that section. I'm still not quite sure about that part. 
**DNM until that is resolved.**

File: Planets and Systems/Harr'qac.md (hunk 4 above)
Comment (Raptor1628) [reply Planets and Systems/Harr'qac.md]: Okay, the way you had it written seemed to suggest that, like other universities, a city built around it over time. If you're suggesting the university also had a city built around it during construction, make that clear.

**Comment by samo-novak** [comment]
Changed the sentence to this:
`It was not originally a part of any city state, but a city was built around it during its construction.`
@Raptor1628

**Comment by Raptor1628** [comment]
*Warble

**Pull request merged**